apps/style/netsim/style.scss

```diff
@@ -473,8 +476,46 @@ svg {
     }
 
     .send_widget_footer {
-      text-align: right;
       margin: 10px;
+      overflow: auto;
+
+      .right-side-controls {
+        float: right;
+        text-align: right;
+      }
+
+      .netsim_packet_size_control {
+        label {
+          display: inline-block;
+        }
+        .slider_inline_wrap {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector `slider_inline_wrap` should be written in lowercase with hyphens